impeller/renderer/backend/vulkan/allocator_vk.cc

```diff
@@ -88,64 +88,47 @@ AllocatorVK::AllocatorVK(std::weak_ptr<Context> context,
                          const vk::PhysicalDevice& physical_device,
                          const std::shared_ptr<DeviceHolder>& device_holder,
                          const vk::Instance& instance,
-                         PFN_vkGetInstanceProcAddr get_instance_proc_address,
-                         PFN_vkGetDeviceProcAddr get_device_proc_address,
                          const CapabilitiesVK& capabilities)
     : context_(std::move(context)), device_holder_(device_holder) {
   TRACE_EVENT0("impeller", "CreateAllocatorVK");
-  vk_ = fml::MakeRefCounted<vulkan::VulkanProcTable>(get_instance_proc_address);
-
-  auto instance_handle = vulkan::VulkanHandle<VkInstance>(instance);
-  if (!vk_->SetupInstanceProcAddresses(instance_handle)) {
```

Comment:
Man, did these even do anything?  These tripped me up, I made them load in my injected proc and they had zero effect.  It's like they load the procs and do nothing with them.

Reply:
I knew I hit a debug assert when I messed up not resolving them earlier. About their use, I can't say.